workers/unity/Assets/Fps/Scripts/GameLogic/StateMachine/Session/SessionInitState.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Resources;
+using Improbable.Worker.CInterop.Alpha;
+using UnityEngine;
+
+namespace Fps
+{
+    public class SessionInitState : SessionState
+    {
+        private readonly SessionScreenController sessionScreenController;
+
+        public SessionInitState(ScreenUIController controller, ConnectionStateMachine owner) : base(controller, owner)
+        {
+            sessionScreenController = controller.FrontEndController.SessionScreenController;
+        }
+
+        public override void StartState()
+        {
+            sessionScreenController.browseButton.enabled = false;
+            sessionScreenController.quickJoinButton.enabled = false;
+            Controller.FrontEndController.SwitchToSessionScreen();
+            var listDeploymentsState = new PrepareDeploymentsListState(Controller, Owner);
+            var getPitState = new GetPlayerIdentityTokenState(listDeploymentsState, Controller, Owner);
+
+            var textAsset = Resources.Load<TextAsset>("DevAuthToken");
+            if (textAsset != null)
+            {
+                Owner.DevAuthToken = textAsset.text.Trim();
+            }
+            else
+            {
+                throw new MissingManifestResourceException(
+                    "Unable to find DevAuthToken.txt inside the Resources folder. Ensure to generate one.");
```

Comment:
`the Resources folder` -> `your Resources folder.` ?
Since we allow the user to configure this